### PR TITLE
feat: configuring fdp-contracts params via options

### LINKED ISSUE #51: Lacking of Blockchain environment config
`fdpStorage` field is initialized with the defaults for reaching out RPC server for blockchain connection and ENS.
There should be a set of environments (currently `Goerli` and `fdp-play`) in form of an enum flag that provides defaults for the granular configuration (also should be possible to set by flags, at least the RpcUrl) of the `ensOptions` of the FdpStorage instance.

### LINKED ISSUE #58: Change copyright info
https://github.com/fairDataSociety/fdp-cli/blob/master/LICENSE

Change from ` Copyright [yyyy] [name of copyright owner]` to `Copyright 2022 "Fair Data Society"`

### LINKED ISSUE #61: `bee-debug-api-url` option is not recognized
Strangely if I set the `bee-debug-api-url` option, the application still tries to fetch stamps on localhost.
Additionally, pod listing shouldn't require stamp listing.
```
$ fdp-cli pod list --bee-api-url http://office-pi:1633 --bee-debug-api-url http://office-pi:1635
ERROR request to http://127.0.0.1:1635/stamps failed, reason: connect ECONNREFUSED 127.0.0.1:1635

There may be additional information in the Bee logs.
```

### PR DESCRIPTION
**Pull request by IgorShadurin**
Close #51
Close #58
Close #61

Before merge it would be great to fix an issue of `fdp-contracts` described here https://github.com/fairDataSociety/fdp-cli/issues/51 (FIXED), then release it, then update `fdp-storage` and release it, then update `fdp-cli` with the latest `fdp-storage`.

### CODE REVIEW

File: test/unit/config.spec.ts [hunk 1]
```diff
@@ -33,28 +35,33 @@ describeCommand('Config data', () => {
...
         data: { beeApiUrl, beeDebugApiUrl },
+        message: 'Config error: `ensNetwork` is not defined or empty',
+      },
+      {
+        data: { beeApiUrl, beeDebugApiUrl, ensNetwork },
         message: 'Config error: `accounts` is not an object',
       },
       {
-        data: { beeApiUrl, beeDebugApiUrl, accounts: 'hello' },
+        data: { beeApiUrl, beeDebugApiUrl, ensNetwork, accounts: 'hello' },
```
Comment (nugaon) [inline test/unit/config.spec.ts:45]: you could rewrite these test records in a way to not modify it after every config structure change in this way
```suggestion
        data: { ...correctConfig, accounts: 'hello' },
```

**Review by nugaon — APPROVED** [review]
LGTM

File: test/unit/config.spec.ts (hunk 1 above)
Comment (IgorShadurin) [reply test/unit/config.spec.ts:45]: the absence of each parameter checks in turn. If a valid object is passed, the validation will not work as intended.

**Review by nugaon — APPROVED** [review]
Let's merge this one : )

**Pull request merged**